lib/agent.js

```diff
@@ -99,6 +99,10 @@ Agent.prototype.start = function (opts) {
     logger.error('Elastic APM isn\'t correctly configured: Missing appName')
     this.active = false
     return this
+  } else if (!/^[a-zA-Z0-9 _-]+$/.test(this.appName)) {
+    logger.error('Elastic APM isn\'t correctly configured: appName contained invalid characters! (allowed: a-z, A-Z, 0-9, _, -, <space>)')
```

Comment:
Could you also log the appName itself here? Something like `appName "in_valid" contains invalid characters.` This could make it easier for the users to see what he actually has set or in case the logs are read by a third paryt, see what it is set to.

I would use `contains` instead of `contained` as at the time of logging, it is still the case.

Reply:
Fixed 👍